src/api/comms.test.js

```diff
@@ -32,6 +43,80 @@ describe('getHeaders', () => {
   });
 });
 
+describe('getPatchHeaders', () => {
+  it('returns default headers when called with no params', () => {
+    expect(getPatchHeaders()).not.toBeNull();
+  });
+
+  it('combines custom headers with the default', () => {
+    const customHeaders = {
+      'X-Foo': 'Bar'
+    };
+    const result = getPatchHeaders(customHeaders);
+    expect(result).toMatchObject(customHeaders);
+    expect(result).toMatchObject(getPatchHeaders());
+  });
+});
+
+describe('patchAddSecretBody', () => {
+  it('patchAddSecretBody', () => {
+    const secretName = 'Groot';
```

Comment:
👽 

Can you change the names in both the describe and the test name itself to be more... "x should do y"?

Reply:
Done 😸